db/forward_iterator.cc

```diff
@@ -441,7 +460,9 @@ void ForwardIterator::SeekInternal(const Slice& internal_key,
         l0_iters_[i]->Seek(internal_key);
       }
 
-      if (!l0_iters_[i]->status().ok()) {
+      if (l0_iters_[i]->status().IsTryAgain()) {
+        continue;
```

Comment:
Assert ```!seek_after_async_io``` here, since we don't expect any more async IO in the second pass.

Reply:
'seek_after_async_io`  is basically indicating it's a second pass. Let me change the variable name as it seems confusing.